tests/test-circular-layout-computation.js

```diff
@@ -0,0 +1,99 @@
+require(['jquery', 'BPTree', 'Empress'], function($, BPTree, Empress) {
+    $(document).ready(function() {
+        // Setup test variables
+        // Note: This is ran for each test() so tests can modify bpArray without
+        // effecting other test
+        module('Circular Layout Computation' , {
+            setup: function() {
+                var tree = new BPTree(
+                    new Uint8Array([1, 1, 1, 0, 1, 0, 0, 1, 0, 0]));
+                var layoutToCoordSuffix = {"Circular": "c1"};
+                var treeData = {
+                    1: {
+                        "color":[1.0, 1.0, 1.0],
+                        "xc0": -2,
+                        "yc0": 2,
+                        "xc1": -2,
+                        "yc1": 0,
+                        "visible": true
+                    },
+                    2: {
+                        "color":[1.0, 1.0, 1.0],
+                        "xc0": 2,
+                        "yc0": 2,
+                        "xc1": 2,
+                        "yc1": 0,
+                        "visible": true
+                    },
+                    3: {
+                        "color":[1.0, 1.0, 1.0],
+                        "xc0": 0,
+                        "yc0": 1,
+                        "xc1": 0,
+                        "yc1": -1,
+                        "arcx0": 2,
+                        "arcy0": 0,
+                        "arcstartangle": 0,
+                        "arcendangle": Math.PI,
+                        "visible": true
+                    },
+                    4: {
+                        "color":[1.0, 1.0, 1.0],
+                        "xc0": 0,
+                        "yc0": -3,
+                        "xc1": 0,
+                        "yc1": -1,
+                        "visible": true
+                    },
+                    5: {
+                        "color":[1.0, 1.0, 1.0],
+                        "xc0": 0,
+                        "yc0": -1,
+                        "xc1": 0,
+                        "yc1": -1,
+                        "visible": true
+                    }
+
+                }
+                this.empress = new Empress(tree, treeData, null,
+                    layoutToCoordSuffix, "Circular", null, null);
+                this.empress._drawer = new Object();
+                this.empress._drawer.VERTEX_SIZE = 5;
+            },
+
+            teardown: function() {
+            }
+        });
+
+        test('Test Circular Layout Arc Computation', function() {
+            // converting result to Set makes validation eaiser since
+            // getObjservationUnionForSamples uses a Set and then converts the
+            // Set to an array. The convertion does not keep order so converting
+            // the result back to a Set makes validation easier.
```

Comment:
I think this comment is out of date.

```suggestion
            // converting result to Set makes validation eaiser since
            // getObjservationUnionForSamples uses a Set and then converts the
            // Set to an array. The convertion does not keep order so converting
            // the result back to a Set makes validation easier.
```

Reply:
This is a comment from an older test file. I copied one of the test files to use as a template and must have not erased all the code in it.